scripts/site/utils/generate-demo.js

```diff
@@ -32,20 +65,23 @@ function generateDemoModule(content) {
     imports += `import { ${declareComponents.join(', ')} } from './${key}';\n`;
     declarations += `\t\t${declareComponents.join(',\n\t')},\n`;
   }
-  imports += `import { NzDemo${componentName(component)}ZhComponent } from './zh.component';\n`;
-  imports += `import { NzDemo${componentName(component)}EnComponent } from './en.component';\n`;
-  declarations += `\t\tNzDemo${componentName(component)}ZhComponent,\n`;
-  declarations += `\t\tNzDemo${componentName(component)}EnComponent,\n`;
-  if (content.pageDemo) {
-    imports += `import { NzPageDemo${componentName(component)}ZhComponent } from './zh.page.component';\n`;
-    imports += `import { NzPageDemo${componentName(component)}EnComponent } from './en.page.component';\n`;
-    declarations += `\t\tNzPageDemo${componentName(component)}ZhComponent,\n`;
-    declarations += `\t\tNzPageDemo${componentName(component)}EnComponent,\n`;
+  if (!content.standalone) {
```

Comment:
For standalone demo, these components are imported in the demo-routes.ts template